hasher-matcher-actioner/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -95,14 +97,19 @@ def raw_lookup():
      * Signal value (the hash)
      * Optional list of banks to restrict search to
     Output:
-     * List of matching content items
+     * List of matching with content_id and, if included, distance values
     """
     signal = require_request_param("signal")
     signal_type_name = require_request_param("signal_type")
-    return lookup_signal(signal, signal_type_name)
+    include_distance = bool(request.args.get("include_distance", False)) == True
```

Comment:
This works for me as a solution, though I think this bool parsing is different than what we do for other flags.

blocking: Can you include it in the list of inputs in the docstring?